### ISSUE TITLE
Headers should have extra spacing away from previous text

### ISSUE THREAD

**Issue by TelepathicGrunt** [neoforged/project-listing#5]
Take a look at the body text of this page
https://projects.neoforged.net/neoforged/neogradle
![image](https://github.com/neoforged/project-listing/assets/40846040/30aa89e8-6fe8-4584-990c-fc5ca99d648c)

It's especially bad here were we have several section separated by only headers. But the headers are too close to the previous text so it all just bleeds together and looks like a giant wall of text that is hard to follow.

Using just 1 rem margin top before each main elements makes the page significantly cleaner
![image](https://github.com/neoforged/project-listing/assets/40846040/52246b2f-f939-4b47-96bc-d949ee7b3927)

**Comment by sciwhiz12** [comment]
Alternatively, the changes in PR #8 (using GitHub-flavored markdown styling) would also fix this issue.

![Image of the "Advanced Settings" section of the NeoGradle project information page with GitHub-flavored markdown styling](https://github.com/user-attachments/assets/02acc68e-ee0a-4dff-8648-cb8838725164)
